src/AutorestSwift/Code Generator/SwiftGenerator.swift

```diff
@@ -62,6 +64,7 @@ class SwiftGenerator: CodeGenerator {
             template: "EnumerationFile",
             toSubfolder: .models,
             withFilename: "Enumerations",
+            withTargetName: targetName,
```

Comment:
I'm not clear why target name is propagated throughout all these methods. I'm not really sure what it is even supposed to be.